pywr/nodes.py

```diff
@@ -746,4 +746,86 @@ def __init__(self, model, name, nodes, **kwargs):
         super(AggregatedNode, self).__init__(model, name, **kwargs)
         self.nodes = nodes
 
+class Transfer(Node):
```

Comment:
If we can think of something better, yes. Transfer doesn't really describe what it does either, just what we've used it for.

ZeroStorageLink?
NullStorage?
BreakLink?